src/ui/components/Card/Card.scss

```diff
@@ -45,9 +46,7 @@
   a:visited {
     color: $link-color;
     display: block;
-    font-variant: all-small-caps;
     font-weight: normal;
-    padding: 10px;
```

Comment:
Cool, up to you if you want to do it in this PR or just file an issue for it and do it outside of here, hence my not-explicit-approve-or-request-changes review.

If you want to file the issue and do it later I'm r+ though. Otherwise I'll have another look once it's finished.